src/game_api/state.hpp

```diff
@@ -53,7 +54,7 @@ struct StateMemory
     int32_t i70;
     uint8_t theme;
     uint8_t theme_next;
-    uint8_t b72;
+    uint8_t win_state; // 0 = no win 1 = tiamat win 2 = hundun win 3 = CO win; set this and next doorway leads to victory scene

```

Comment:
Does it make sense to add an enum for the Lua API for this?

Reply:
Good point, I'll add it